technology/kotlin-notebook/school-timetabling.ipynb

```diff
@@ -0,0 +1,641 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "source": [
+    "# School timetabling kotlin notebook\n",
+    "\n",
+    "This Kotlin Notebook solves the school timetabling problem with [Timefold](https://timefold.ai], the open source planning solver AI.\n",
+    "\n",
+    "![School timetabling input output](https://timefold.ai/docs/timefold-solver/latest/_images/quickstart/school-timetabling/schoolTimetablingInputOutput.png)\n",
+    "\n",
+    "## Dependencies\n",
+    "\n",
+    "Add the Timefold solver dependency:"
+   ],
+   "metadata": {
+    "collapsed": false
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 74,
+   "outputs": [],
+   "source": [
+    "@file:DependsOn(\"ai.timefold.solver:timefold-solver-core:1.3.0\")"
```

Comment:
At this point on `development` branch, this version should be `999-SNAPSHOT`.
Please go into `.github/workflows/release.yml` and add support there for bumping the version during the release.

Reply:
Agreed. Alternatively, we can explore support for `%use timefold` to avoid the versioning alltogheter (which is debatable wheter that is a good thing).